ilastik.py

```diff
@@ -77,6 +77,7 @@ def main():
     # Provide your command-line args here. See examples below.
     
     ## Auto-open an existing project
+    parsed_args.project='/home/andreas/MyProject.ilp'
```

Comment:
I guess, one line would be nice as an example but more are not needed. If you would like to remove it, the nice2know.rst file contains some information on the as well.